hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/schedule/ScheduleEqualityVirtualValue.java

```diff
@@ -135,14 +135,23 @@ public void serialize(SerializableDataOutputStream out) throws IOException {
 
     @Override
     public void serialize(ByteBuffer out) throws IOException {
+        serializeReturningBytesWritten(out);
+    }
+
+    public int serializeReturningBytesWritten(ByteBuffer out) {
```

Comment:
Should it be public or can be changed to package-private?

Reply:
Fixed